Source/WebCore/page/Navigation.cpp

```diff
@@ -725,22 +739,22 @@ bool Navigation::dispatchTraversalNavigateEvent(HistoryItem& historyItem)
     // FIXME: Set destinations state
     Ref destination = NavigationDestination::create(historyItem.url(), WTFMove(destinationEntry), isSameDocument);
 
-    return innerDispatchNavigateEvent(NavigationNavigationType::Traverse, WTFMove(destination), { });
+    return innerDispatchNavigateEvent(NavigationNavigationType::Traverse, WTFMove(destination), { }, nullptr);
```

Comment:
same as before, maybe we can use the default argument instead of explicitly passing `nullptr`.

Reply:
Done.